### ISSUE TITLE
wishlist: crm conf primitive complains should complain that a package not installed

### ISSUE THREAD

**Issue by aleksei-burlakov** [ClusterLabs/crmsh#967]
When doing
`crm configure primitive st-ssh stonith:external/ssh params hostlist=node1`
the crm would complain
`ERROR: st-ssh: parameter "hostlist" is not known`
(because the `libglue-devel` containing `stonith:external/ssh` was not installed). The user would rather get the message that `stonith:external/ssh` or `libglue-devel` was not installed, than that the `parameter "hostlist" is not known`.

**Comment by liangxin1300** [comment]
@aleksei-burlakov Thanks for pointing this
IMO, maybe crmsh can maintain a map for most used RA, to show which package name that this RA is belonged?
OTOH, does resource-agent already have such a map? 
@zzhou1 @gao-yan What do you think? Good to append the feature list for 15sp5?

**Comment by zzhou1** [comment]
Agree, this Use case is valid to improve User Experience. From the design wise, the code could be generic and flexible to avoid maintenance burden if any. We could add it to the backlog to do further assessment if possible make it to the next release.

**Comment by liangxin1300** [comment]
>  OTOH, does resource-agent already have such a map?
I think it's hard for resource-agent to maintain such a `RA-name:package-name` map, since for different vendor the package name is different

**Comment by aleksei-burlakov** [comment]
Maybe, just tell the user the package is not installed, and let him find out which one himself.

**Comment by liangxin1300** [comment]
I think for most of RA, `exitreason` from crm_mon can give more details

Although not for this stonith:external/ssh RA:
```
* st-ssh_start_0 on hanode1 'error' (1): call=26, status='complete', last-rc-change='Mon Nov 14 09:11:19 2022', queued=0ms, exec=1226ms
```
